great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -661,7 +664,7 @@ def _build_datetime_columns_rule() -> Rule:
             "mostly": 1.0,
             "strict_min": False,
             "strict_max": False,
-            "bins": 10,
+            "allow_relative_error": "linear",
```

Comment:
This parameter seems to be important, but why can't this just be added to `column.histogram`?

Reply:
@NathanFarmer Because it would contravene the original API, whereby the `column.histogram` metric accepts the list, containing the bin boundaries (the `bins` parameter).  So I had to revert in order to stay true to the original intent.  Thanks!